dicegame/runner.py

```diff
@@ -2,54 +2,78 @@
 from .utils import i_just_throw_an_exception
 
 class GameRunner:
+    """
+    Hosts games of Sum the Dice.
+
+    Attributes:
+        round: An integer indicating the current round.
+        wins: An integer indicating how many times the user has won in total.
+        losses: An integer indicating how many times the user has lost in total.
+        c: An integer indicating the number of consecutive wins the user has
+           right now.
+    """
+
+    # Constants
+    N_DICE = 5          # Number of dice to roll
+    N_TO_WIN = 6        # Consecutive wins required to win
 
     def __init__(self):
-        self.dice = Die.create_dice(5)
         self.reset()
 
     def reset(self):
+        """Reset all variables to 0."""
         self.round = 1
         self.wins = 0
-        self.loses = 0
+        self.losses = 0
+        self.c = 0
+        self.roll()
+
+    def roll(self):
+        self.dice = Die.create_dice(self.N_DICE)
 
     def answer(self):
+        """Returns the current accurate dice sum."""
         total = 0
         for die in self.dice:
             total += die.value
         return total
+    
+    def play_round(self):
+        """Rolls one set of dice and ask for results.
+        Requires user input of an integer."""
+        self.roll()
+
+        for die in self.dice:
+            print(die.show())
+
+        # TODO (XS - 4/4/23) can't handle a non-int guess

```

Comment:
Seems like this would be a good time to fix it - worth doing a try/catch here and looping like we do in lines 82-83?